src/ClientConfig.cc

```diff
@@ -416,19 +428,31 @@ bool ClientConfig::LoadConfig(const std::string &_file)
   std::string homePath;
   gz::common::env(IGN_HOMEDIR, homePath);
   std::string cacheLocation = gz::common::joinPaths(
-    homePath, ".ignition", "fuel");
+    homePath, ".gz", "fuel");
```

Comment:
Let's just change the env var without changing the directory in this PR. It's safer to update the directory together with https://github.com/gazebosim/gz-fuel-tools/pull/250.

Reply:
Reverted all instances in fuel-tools